base/src/main/java/io/spine/code/proto/ColumnOption.java

```diff
@@ -106,20 +103,11 @@ public static boolean isColumn(FieldDeclaration field) {
     }
 
     /**
-     * Returns {@code true} if the given message type is eligible for having columns.
-     *
-     * <p>Currently, only projection and process manager types can have columns.
-     *
-     * <p>The column presence for {@linkplain io.spine.option.EntityOption.Kind#ENTITY generic}
-     * entity types is also allowed for convenience of tests.
+     * Returns {@code true} if the given message type is declared as entity and may have columns.
      */
     @SuppressWarnings("BooleanMethodIsAlwaysInverted") // For readability.
-    private static boolean eligibleForColumns(MessageType messageType) {
+    private static boolean declaresAsEntity(MessageType messageType) {
```

Comment:
I suggest `declaresEntity(..)` for better readability. Or, citing the doc, `declaredAsEntity(..)`.